src/app/components/Statistics/DateRange/DateRange.tsx

```diff
@@ -109,6 +141,14 @@ const DateRange = ({ initialStartDate, InitianEndDate, setEndDate, setStartDate,
         </CustomInputContainer>
     ))
 
+    const getMaxDate = () => {
+        if (!disableFuture) {
```

Comment:
Swapping a condition here would allow you to delete `else` section entirely, if `disableFuture` => return new Date(), if not, do nothing (so it will return undefined by default)

Reply:
Solved :+1: